### ISSUE TITLE
Convert examples to IPython notebooks

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo#53]
When you convert one, check it off of the list! Or, if it can't be converted, check it off and put a comment below. If an example is waiting on some key functionality, make an issue and link it below (as in network_array_2d.py).

- [x] 2drepresentation.py
- [x] addition.py
- [ ] armcontrol.py (requires simulation visualizer)
- [x] basalganglia.py
- [x] combining.py
- [x] communication.py
- [x] controlledintegrator.py
- [x] controlledintegrator2.py
- [x] controlledoscillator.py
- [x] convolve.py (requires SPA knowledge)
- [x] integrator.py
- [x] learn_communicate.py
- [x] learn_product.py
- [x] learn_square.py 
- [x] manyneurons.py
- [x] matrix_multiplication.py
- [x] multiplication.py
- [x] network_array_2d.py
- [x] oscillator.py
- [x] question_control.py
- [x] question_memory.py (requires SPA knowledge)
- [x] question.py (requires SPA knowledge)
- [x] singleneuron.py
- [x] spa_sequence.py
- [x] spa_sequencerouted.py
- [x] squaring.py
- [x] twoneurons.py
- [ ] vehicle.py (requires simulation visualizer)

**Comment by jgosmann** [comment]
For some reason the basal ganglia example is empty, but it was checked as done above. :open_mouth:

**Comment by Seanny123** [comment]
I did it once upon a time and I must have messed up pushing it. Give me a
day to dig it up.

**Comment by jgosmann** [comment]
Found it in the `docs` branch: a263a995421efbb3066ab36dcdcdc93d6cd97ae8
Will cherry pick it into its own PR.

**Comment by hunse** [comment]
Okay, I think we've done everything except the examples that require complex visualization (`armcontrol.py` and `vehicle.py`). I'm declaring this done.

**Comment by tbekolay** [comment]
:tada: :cake:
